### PR TITLE
Perform case insensitive lookups for precompiled views

### PR DESCRIPTION
**Pull request by pranavkm**
Fixes https://github.com/aspnet/MvcPrecompilation/issues/37

### CODE REVIEW

**Comment by rynowak** [comment]
FYI this is definitely announcment-worthy

**Comment by pranavkm** [comment]
@Eilon good to get this in for preview2?

File: src/Microsoft.AspNetCore.Mvc.Razor/Resources.resx [hunk 1]
```diff
@@ -191,4 +191,7 @@
   <data name="PropertyMustBeSet" xml:space="preserve">
     <value>The property '{0}' of '{1}' must not be null.</value>
   </data>
+  <data name="RazorViewCompiler_ViewPathsDifferOnlyInCase" xml:space="preserve">
+    <value>The following precompiled view paths differ only in case which is not supported: '{0}', '{1}'.</value>
```
Comment (Eilon) [inline src/Microsoft.AspNetCore.Mvc.Razor/Resources.resx]: Add a comma before `which`.

Comment (Eilon) [inline src/Microsoft.AspNetCore.Mvc.Razor/Resources.resx]: And maybe put the two paths on their own line, to make them easier to compare?

```
The following blah blah are blah blah:
1. {0}
2. {1}
```

**Pull request merged**